rimage/pkcs1_5.c

```diff
@@ -68,6 +68,97 @@ static void bytes_swap(uint8_t *ptr, uint32_t size)
  * manifest header (Public Key, Exponent and Signature).
 */
 
+int pkcs_sign_v1_5(struct image *image, struct fw_image_manifest_v1_5 *man,
+		   void *ptr1, unsigned int size1)
+{
```

Comment:
this confusing, you are adding support for manifest v1.5 in a file dedicated to PKCS 1.5...

Reply:
I think pkcs_sign_v1_5 and pkcs_sign_v1_8 can be merge to pkcs_sign, since most of them are the same. @lgirdwood what is your opinion? The different version manifest have a dedicated PKCS version?

Reply:
@zhuyingjiang the confusiion is around similar version naming with manifest and PKCS.  You could rename lime pkcs_v1_5_sign_man_v1_8()

Reply:
OK